docs/hub-deployment-guide/new-cluster/aws.md

```diff
@@ -88,19 +101,21 @@ Once you have a `.jsonnet` file, you can render it into a config file that eksct
 can read.
 
 ```bash
-jsonnet <your-cluster>.jsonnet > <your-cluster>.eksctl.yaml
+jsonnet $CLUSTER_NAME.jsonnet > eksctl-config.yaml
```

Comment:
I would highly recommend keeping the name of the cluster name in the generated yaml file as well, to prevent accidents where you think you are applying one yaml file but are actually applying a file generated from a different jsonnet! *.eksctl.yaml maybe added to .gitignored if needed.